platform/android/.gitignore

```diff
@@ -21,6 +21,7 @@ local.properties
 
 # Token file
 MapboxGLAndroidSDKTestApp/src/main/res/values/developer-config.xml
+MapboxGLAndroidSDKWearTestApp/src/main/res/values/developer-config.xml
```

Comment:
Is the gradle task responsible for generating `MapboxGLAndroidSDKTestApp/src/main/res/values/developer-config.xml` also capable of generating `MapboxGLAndroidSDKWearTestApp/src/main/res/values/developer-config.xml`?